readme.md

```diff
@@ -273,3 +273,29 @@ final Response response = serviceFactory.get(request);
 ```
 
 
+## How do I use Address Submission?
+
+Add the Address Submission dependency:
+```xml
+<dependency>
+	<groupId>com.afrigis.services</groupId>
+	<artifactId>addresssubmission</artifactId>
+	<version>3.0.1</version>
+</dependency>
+```
+
+
+```java
+import com.afrigis.services.AfriGISServices;
+import com.afrigis.services.ServiceCallFactory;
+import com.afrigis.services.geocode.AddressRequest;
+import com.afrigis.services.geocode.AddressResponse;
+
+ServiceCallFactory serviceFactory = AfriGISServices.instance("Key","Secret");
+AddressSubmissionRequest addressSubParams = new AddressSubmissionRequest("KEY", "Address", "AdditionalInfomation");
+AddressSubmissionResponse response = serviceFactory.get(addressSubParams);
+List<SubmissionResult> addressSubmList = response.listResults();
+System.out.println (addressSubmList.get(0).getClientID());
```

Comment:
Please explicitly extract the SubmissionResult instance and demonstrate using it:
```
SubmissionResult result = addressSubmList.get(0);
System.out.println (result.getClientID());